integration-tests/tests/test_identity_transformation.py

```diff
@@ -135,10 +140,11 @@ def test_clp_s_identity_transform(
 
 
 def _clp_s_compress_and_decompress(
-    integration_test_config: IntegrationTestConfig, test_paths: CompressionTestConfig
+    core_config: CoreConfig,
+    test_paths: CompressionTestConfig,
```

Comment:
Hm, do you know why the formatter didn't put these on a single line? I feel like that's what black would do.

Reply:
I think it's due to my extra comma.

Reply:
Also, I've set up the linting process so that we don't run both `black` and `ruff` for a single directory. I think it's `ruff` for `integration-tests`